scripts/qa_crowbarsetup.sh

```diff
@@ -3346,11 +3346,17 @@ function oncontroller_run_tempest
     fi
     if [ -n "$ostestroptions" ]; then
         zypper -n in python-os-testr
+        if [[ $want_magnum = 1 ]]; then
+            pushd /var/lib/openstack-magnum-test
```

Comment:
so we're switching from tempest to magnum tests? why isn't this done as a cct feature plugin like other functional tests?